sdk/src/common/platform/fork_windows.cc

```diff
@@ -0,0 +1,20 @@
+#include "src/common/platform/fork.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace common
+{
+namespace platform
+{
+int AtFork(void (*prepare)(), void (*parent)(), void (*child)()) noexcept
+{
+  (void)prepare;
+  (void)parent;
+  (void)child;
+  return 0;
+}
+}  // namespace platform
+}  // namespace common
+}  // namespace sdk
+OPENTELEMETRY_BEGIN_NAMESPACE
```

Comment:
END :)

C:\Users\circleci\project\sdk\src\common\platform\fork_windows.cc(20): error C1075:  '{': no matching token found

Reply:
Ah, that's what I missed. Fixed. Thanks.

Reply:
I switched to FastRandomNumberGenerator.

It takes up only 16 bytes and here's the benchmark comparison
https://2145-193156399-gh.circle-artifacts.com/0/benchmark/sdk/test/common/random_benchmark_result.txt
```
2020-03-27 02:18:54
Running bazel-out/host/bin/sdk/test/common/random_benchmark
Run on (36 X 3370.14 MHz CPU s)
CPU Caches:
  L1 Data 32K (x18)
  L1 Instruction 32K (x18)
  L2 Unified 1024K (x18)
  L3 Unified 25344K (x1)
Load Average: 6.47, 9.48, 13.46
-------------------------------------------------------------------
Benchmark                         Time             CPU   Iterations
-------------------------------------------------------------------
BM_RandomIdGeneration          2.13 ns         2.13 ns     63421653
BM_RandomIdStdGeneration       6.18 ns         6.18 ns     22676378
```
The algorithm [xorshift](https://en.wikipedia.org/wiki/Xorshift) is fine for ID generation.